frontend/src/metabase/visualizations/visualizations/SmartScalar/utils.ts

```diff
@@ -63,3 +79,244 @@ export const getValueHeight = (height: number): number => {
 export const getChangeWidth = (width: number): number => {
   return Math.max(width - ICON_SIZE - ICON_MARGIN_RIGHT - 2 * SPACING, 0);
 };
+
+export const COMPARISON_TYPES = {
```

Comment:
@kulyk You'll want to add the new comparison type here `CUSTOM_VALUE: "customValue"`